src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
 body {
   margin: 0;
 }
```

Comment:
If you did it inside `reset.scss` you can remove this rule here.

Reply:
Changed